pkg/metadata/inventories/inventories.go

```diff
@@ -280,7 +280,7 @@ func initializeConfig(cfg config.Config) {
 	SetAgentMetadata(AgentConfigProxyHTTP, clean(cfg.GetString("proxy.http")))
 	SetAgentMetadata(AgentConfigProxyHTTPS, clean(cfg.GetString("proxy.https")))
 	SetAgentMetadata(AgentCWSEnabled, config.Datadog.GetBool("runtime_security_config.enabled"))
-	SetAgentMetadata(AgentProcessEnabled, config.Datadog.GetBool("process_config.enabled"))
+	SetAgentMetadata(AgentProcessEnabled, config.Datadog.GetBool("process_config.process_collection.enabled"))
```

Comment:
Should we had a new field for `process_config.container_collection.enabled` ? This payload is ingested by the inventory product and allow users to known all the agent using a specific feature for example.

Reply:
I think that makes sense, would anything need to change on the backend to support this?

Reply:
No change needed in the backend. I'll notify the inventory team once this is done that a new value is being sent.